setup.py

```diff
@@ -233,7 +234,7 @@ def run(self):
     extras["flax"] = []  # jax is not supported on windows
 else:
     extras["retrieval"] = deps_list("faiss-cpu", "datasets")
-    extras["flax"] = deps_list("jax", "jaxlib", "flax")
+    extras["flax"] = deps_list("jax", "jaxlib", "flax", "optax")
```

Comment:
optax is required for any flax training and is very lightweight. I think it is ok to put it here, no?

Reply:
@LysandreJik @sgugger @patil-suraj

Reply:
Fine by me too.